popSimManu.tex

```diff
@@ -166,8 +165,8 @@ \section*{Introduction}
 methods using bespoke simulations. 
 Often these studies focus more on showcasing a novel method than on
 rigorously comparing it with competing methods.
-Moreover, this approach results in a great deal of duplicated effort across research
-groups, and contributes to decreased reproducibility and transparency.
+Moreover, this results in a great deal of duplicated effort,
```

Comment:
unclear to what "this" refers back to in this context. I'd keep it "this approach"

Reply:
it's not an "approach"; that's the problem. Changing to "this situation".